src/js/components/add-journal-entry/add-journal-entry.js

```diff
@@ -27,21 +38,24 @@ class AddJournalEntry extends MaharaBaseComponent {
         }
         return <section>
             <h1>{pageTitle}</h1>
-            <JournalEntry parent={this} {...this.props} {...journalParams} ref="journalEntry"/>
+            <JournalEntry parent={this} {...this.props} {...journalParams} ref="journalEntry" onChangeJournal={this.changeJournal}/>
```

Comment:
what is parent used for? Can we get rid of it? Usually it's not a good idea for child to have reference to parent.